internal/jimm/model.go

```diff
@@ -1347,3 +1351,67 @@ func (j *JIMM) ChangeModelCredential(ctx context.Context, user *openfga.User, mo
 
 	return nil
 }
+
+// ListModels list the models that the user has access to. It intentionally excludes the
+// controller model as this call is used within the context of login and register commands.
+func (j *JIMM) ListModels(ctx context.Context, user *openfga.User) ([]base.UserModel, error) {
+	const op = errors.Op("jimm.ListModels")
+	zapctx.Info(ctx, string(op))
+
+	// Get models uuids user has access to
+	uuids, err := user.ListModels(ctx, ofganames.ReaderRelation)
+	if err != nil {
+		return nil, errors.E(op, err, "failed to list models")
+	}
+
+	// Get the models themselves
+	models, err := j.DB().GetModelsByUUID(ctx, uuids)
```

Comment:
Mhmmm... You're thinking discover the models from the controllers?

Reply:
no, we just need to fetch models one by one and fail gracefully if a model cannot be found in the db. it might be because of a leftover tuple in openfga.

Reply:
As discussed, resolving.